src/Views/Form/Templates/Sequoia/assets/js/form.js

```diff
@@ -154,6 +154,11 @@
 				//Setup input icons
 				setupInputIcon( '#give-first-name-wrap', 'user' );
 				setupInputIcon( '#give-email-wrap', 'envelope' );
+
+				setupGatewayIcon( 'manual', 'fas fa-tools' );
+				setupGatewayIcon( 'offline', 'fas fa-wallet' );
+				setupGatewayIcon( 'paypal', 'gateway-icon paypal' );
+				setupGatewayIcon( 'stripe', 'far fa-credit-card' );
```

Comment:
@henryholtgeerts what's the fallback icon for other credit card gateways?

Reply:
I had imagined this would be handled in an addon compatibility issue, but I think the fallback would be the same fontawesome credit card icon. I'll add some fallback functionality to handle cases where a gateway is listed but no icon is defined.